packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/imageEditors/Resizer.ts

```diff
@@ -222,4 +244,4 @@ const setHandleStyle: Record<
         `position:relative;width:${RESIZE_HANDLE_SIZE}px;height:${RESIZE_HANDLE_SIZE}px;background-color: ${borderColor};cursor:${direction}-resize;${topOrBottom}:-${RESIZE_HANDLE_MARGIN}px;${leftOrRight}:-${RESIZE_HANDLE_MARGIN}px;`,
     1: (direction, leftOrRight, topOrBottom) =>
         `position:relative;width:${RESIZE_HANDLE_SIZE}px;height:${RESIZE_HANDLE_SIZE}px;background-color: #FFFFFF;cursor:${direction}-resize;${topOrBottom}:-${RESIZE_HANDLE_MARGIN}px;${leftOrRight}:-${RESIZE_HANDLE_MARGIN}px;border-radius:100%;border: 2px solid #bfbfbf;box-shadow: 0px 0.36316px 1.36185px rgba(100, 100, 100, 0.25);`,
-};
+};
```

Comment:
let's don't remove this CR at the end of file

Reply:
Sure. But I'm a little curious about why we don't enable tslint rule "eofline"?